apps/site/components/Downloads/ChangelogModal/index.stories.tsx

```diff
@@ -180,10 +179,7 @@ export const Default: Story = {
     open: false,
     heading: 'Node v18.17.0',
     subheading: "2023-07-18, Version 18.17.0 'Hydrogen' (LTS), @danielleadams",
-    avatars: names.map(name => ({
-      src: getGitHubAvatarUrl(name),
-      alt: name,
-    })),
+    avatars: names,
```

Comment:
For strange rease there are username that display fallback instead of GitHub profile picture (it's valid username)